app-be/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.0.3
\ No newline at end of file
+sbt.version = 0.13.16
```

Comment:
@k3nda FYI - necessary due to the SbtWeb plugin, which is not compatible with version 1+...